server/manager/src/main/java/org/apache/accumulo/master/tableOps/create/CreateTable.java

```diff
@@ -78,7 +79,11 @@ public long isReady(long tid, Master environment) throws Exception {
   }
 
   @Override
-  public void undo(long tid, Master env) {
+  public void undo(long tid, Master env) throws Exception {
+    // Clean up split files if create table operation fails
+    Path p = tableInfo.getSplitPath().getParent();
+    FileSystem fs = p.getFileSystem(env.getContext().getHadoopConf());
+    fs.delete(p, true);
```

Comment:
returns boolean, should we log when delete fails?

Reply:
I can add a log for it if it is necessary but I should also add a check to make sure there is even a split file to begin with... I will add that.